build/ci/pr/Jenkinsfile

```diff
@@ -13,6 +13,9 @@ pipeline {
         VAULT_ROLE_ID = credentials('vault-role-id')
         VAULT_SECRET_ID = credentials('vault-secret-id')
         GCLOUD_PROJECT = credentials('k8s-operators-gcloud-project')
+        REGISTRY = "eu.gcr.io"
```

Comment:
My problem with that is that to repro this job I need to copy each of those vars, export them and run `./setenv`. Then find deployer-config snipper and run it too. 
Now it's only copy paste a single snippet (.env + deployer-config) from the code. 

Could we just move this to `createConfig` and invoke `./setenv` there too?